python/test/test_github_service.py

```diff
@@ -1269,5 +1272,294 @@ def edit(self, assignees):
         self.assignees = assignees
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
+@patch("gql.Client.__new__", new=MagicMock)
+@patch("python.services.github_service.Github")
+class TestReportOnInactiveUsers(unittest.TestCase):
+
+    def test_report_on_inactive_users(self, mock_github_client_core_api):
+        team1 = Mock()
+        team1.name = "team1"
+
+        teams = {
+            'team1': {
+                'github_team': 'github_team',
+                'remove_from_team': True,
+            }
+        }
+
+        user = Mock()
```

Comment:
Optional: if use the `def setUp(self):` before line 1280, could move alot of these variables that are spread across the below tests into one location and re-use?